src/sentry/search/eap/resolver.py

```diff
@@ -83,7 +83,7 @@ def resolve_meta(self, referrer: str) -> RequestMeta:
         if span:
             span.set_tag("SearchResolver.params", self.params)
         return RequestMeta(
-            organization_id=self.params.organization_id,
+            organization_id=self.params.rpc_organization_id,
```

Comment:
Actually for now can you just check org_id when we call instead? The alternative is I'd prefer any call to organization_id errors and we change the setter to something like `_organization_id`

Reply:
done